doc/install-dependencies.md

```diff
@@ -120,3 +122,29 @@ brew install openresty/brew/openresty etcd luarocks curl git
 # start etcd server with v2 protocol
 etcd --enable-v2=true &
 ```
+
+How to compile the Openresty
```

Comment:
Compiling OpenResty from source is very complicated, it's not easy to make it clear. So I don't think we need to explain how to compile OpenResty, just a link is enough.

Reply:
Agree with this.